dnnlib: Running training.training_loop.training_loop() on localhost...
TF Seed: 659662259
NP Seed: 1000
Streaming data using training.dataset.TFRecordDataset...
Traceback (most recent call last):
  File "train.py", line 233, in <module>
    main()
  File "train.py", line 86, in main
    run(opt)
  File "train.py", line 200, in run
    dnnlib.submit_run(**kwargs)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\dnnlib\submission\submit.py", line 290, in submit_run
    run_wrapper(submit_config)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\dnnlib\submission\submit.py", line 242, in run_wrapper
    util.call_func_by_name(func_name=submit_config.run_func_name, submit_config=submit_config, **submit_config.run_func_kwargs)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\dnnlib\util.py", line 257, in call_func_by_name
    return func_obj(*args, **kwargs)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\training\training_loop.py", line 149, in training_loop
    training_set = dataset.load_dataset(data_dir=config.data_dir, verbose=True, **dataset_args)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\training\dataset.py", line 240, in load_dataset
    dataset = dnnlib.util.get_obj_by_name(class_name)(**adjusted_kwargs)
  File "C:\Users\admin\Desktop\1.11\OroJaR-master\progan_experiments\training\dataset.py", line 102, in __init__
    assert all(lod in tfr_lods for lod in range(self.resolution_log2 - 1))
AssertionError
